### PR TITLE
improve getting started on arbitrary clusters

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

File: python/hail/docs/getting_started.rst [hunk 1]
```diff
@@ -92,56 +92,80 @@ cluster::
...
-    PYTHONPATH="$PYTHONPATH:$HAIL_HOME/build/distributions/hail-python.zip:$SPARK_HOME/python:$SPARK_HOME/python/lib/py4j-*-src.zip" \
-    ipython
+.. code-block:: sh
 
-Within the interactive shell, check that you can initialize Hail by running the
-following commands. Note that you must pass in the existing ``SparkContext``
-instance ``sc`` to the ``hail.init`` function.
+    export SPARK_HOME=/path/to/spark-2.2.0/
+    export HAIL_HOME=/path/to/hail/
+    export PYTHONPATH="$PYTHONPATH:$HAIL_HOME/build/distributions/hail-python.zip"
```
Comment (tpoterba) [inline python/hail/docs/getting_started.rst]: invert these to prepend. PYTHONPATH is stupid and can cause problems if it starts with a colon

Comment (danking) [reply python/hail/docs/getting_started.rst]: Hmm. Is it rude to prepend our stuff on top of what's already there?

Comment (tpoterba) [reply python/hail/docs/getting_started.rst]: usually there's nothing there

Comment (danking) [reply python/hail/docs/getting_started.rst]: Ok I took a look at POSIX and I think we can post-pend in a PYTHONPATH safe way:
```
(hail) dking@wmb16-359 # sh
sh-3.2$ PYTHONPATH=""
sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}:$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ 
sh-3.2$ echo $PYTHONPATH
:/Users/dking/projects/hail/build/distributions/hail-python.zip
sh-3.2$ PYTHONPATH=""
sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ 
sh-3.2$ echo $PYTHONPATH
/Users/dking/projects/hail/build/distributions/hail-python.zip
sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ echo $PYTHONPATH
/Users/dking/projects/hail/build/distributions/hail-python.zip:/Users/dking/projects/hail/build/distributions/hail-python.zip
sh-3.2$ unset PYTHONPATH
sh-3.2$ echo $PYTHONPATH

sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ echo $PYTHONPATH
/Users/dking/projects/hail/build/distributions/hail-python.zip
```
(POSIX parameter substitution)[http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02]

**Pull request merged**